code/game/objects/structures/flora.dm

```diff
@@ -341,3 +341,11 @@
 /obj/structure/flora/sif/eyes/Initialize()
 	icon_state = "[initial(icon_state)][rand(1,3)]"
 	. = ..()
+
+/obj/structure/flora/sif/attack_hand(mob/user)

```

Comment:
Is this meant to be specific to the eye plant? If so, you used the incorrect path. If not, please move the proc upwards into the 301-ish line, to keep it directly after the `structure/flora/sif` definition.

Additionally, the `visible_message` proc has inputs for what the user sees and what everyone else sees, those really ought to be used.